{"name": "Test Add Business Owner", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Locator.wait_for: Timeout 30000ms exceeded.\nCall log:\n  - waiting for locator(\"div[role='option']:has-text('Male')\").first to be visible\n    63 × locator resolved to hidden <div role=\"option\" aria-label=\"Male\" id=\"gender_list_0\" aria-selected=\"false\">male</div>", "trace": "@allure.title(\"Test Add Business Owner\")\n    def test_add_business_owner():\n        with sync_playwright() as p:\n            browser = p.chromium.launch(headless=False)\n            page = browser.new_page()\n    \n            page.goto(\"https://dashboard.impactoapps.com/business-owners\")\n    \n            allure.attach(page.screenshot(), name=\"page_open\", attachment_type=allure.attachment_type.PNG)\n    \n            page.click(\"button:has-text('Add Business Owner')\")\n            page.wait_for_timeout(1000)\n    \n            try:\n                page.wait_for_selector(\"h5:has-text('Add Business Owner')\", timeout=5000)\n            except:\n                page.wait_for_selector(\"text=Add Business Owner\", timeout=5000)\n    \n            page.fill(\"#name\", \"Automation User\")\n            page.fill(\"#email\", generate_random_email())\n            page.fill(\"#countryDialCode\", \"+880\")\n            page.fill(\"#phone\", \"01712345678\")\n            page.fill(\"#dateOfBirth\", \"1990-01-01\")\n    \n            page.click(\"#gender\")\n            page.wait_for_timeout(500)\n>           page.locator(\"div[role='option']:has-text('Male')\").first.wait_for(state=\"visible\")\n\ntest_nm.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:17920: in wait_for\n    self._sync(self._impl_obj.wait_for(timeout=timeout, state=state))\n..\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\_impl\\_locator.py:685: in wait_for\n    await self._frame.wait_for_selector(\n..\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\_impl\\_frame.py:323: in wait_for_selector\n    await self._channel.send(\"waitForSelector\", locals_to_params(locals()))\n..\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\_impl\\_connection.py:61: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <playwright._impl._connection.Connection object at 0x000001D52B021E80>, cb = <function Channel.send.<locals>.<lambda> at 0x000001D52B38FA60>, is_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.wait_for: Timeout 30000ms exceeded.\nE           Call log:\nE             - waiting for locator(\"div[role='option']:has-text('Male')\").first to be visible\nE               63 × locator resolved to hidden <div role=\"option\" aria-label=\"Male\" id=\"gender_list_0\" aria-selected=\"false\">male</div>\n\n..\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\_impl\\_connection.py:528: TimeoutError"}, "attachments": [{"name": "page_open", "source": "6855b00f-c73d-4125-b87e-ffc41cd08761-attachment.png", "type": "image/png"}], "start": 1748857703968, "stop": 1748857744396, "uuid": "980e511e-7ddf-4faf-8c02-375e3a29434c", "historyId": "c14d723a995d644cd63448c427b6928f", "testCaseId": "c14d723a995d644cd63448c427b6928f", "fullName": "test_nm#test_add_business_owner", "labels": [{"name": "suite", "value": "test_nm"}, {"name": "host", "value": "DESKTOP-VKIO4T7"}, {"name": "thread", "value": "12576-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_nm"}]}